src/ucp/core/ucp_worker.h

```diff
@@ -192,6 +193,8 @@ typedef struct ucp_worker {
     ucs_strided_alloc_t           ep_alloc;      /* Endpoint allocator */
     ucs_list_link_t               stream_ready_eps; /* List of EPs with received stream data */
     khash_t(ucp_worker_ep_hash)   ep_hash;       /* Hash table of all endpoints */
```

Comment:
Do we still need this?

Reply:
for now yes, because the ep pointer is not passed yet, will be removed in next PRs